binaries/daemon/src/lib.rs

```diff
@@ -510,7 +510,6 @@ impl Daemon {
         if let Some(token) = drop_token {
             dataflow.check_drop_token(token).await?;
         }
-        let data_bytes = data_bytes;
```

Comment:
We will need this later when we implement distributed deployments, i.e. split the dataflow across multiple `dora-daemon` instances, potentially running on different machines. So I'm in favor of keeping this line so that we remember where we have to fill in this implemenation later. I agree with you that we should silence this warning, though.

How about we replace it with a TODO comment:

```rust
// TODO: Send the data to remote daemon instances if the dataflow
// is split across multiple machines
let _data_bytes = data_bytes;
```

Reply:
I agree with your decision. Thanks!

Reply:
Ok, I applied the above patch.